### DISCUSSION TITLE
Problem with russian language

### DISCUSSION THREAD

**Discussion by Dimensionnnnn** [Alexandre-Fernandez/astro-i18n#78] [General]
Greetings, I have encountered a problem with the Russian language display, I have customized the locales and also customized the paths, so in English everything displays fine, however, when I get to the Russian part I am having trouble getting quality text, I checked my meta tags and I am using utf-8, also if I use sharded Russian text everything also displays correctly, could you please tell me what the problem could be?
<img width="626" alt="image" src="https://github.com/Alexandre-Fernandez/astro-i18n/assets/84111957/42001494-fd58-4d11-86f4-f5d7452fd078">

**Comment by Alexandre-Fernandez** [comment]
Hi can you please provide a repository where I can reproduce this problem ?
Showcase the problem and the expected result.

**Comment by Dimensionnnnn** [comment]
> Hi can you please provide a repository where I can reproduce this problem ? Showcase the problem and the expected result.

Greetings, I found that the problem was using a video as a preloader on my page, the strange thing is that it created this effect, but when I started using just <video /> instead of <video><source></source>text</video> everything went smoothly, I don't know what to do with it, but so far it works this way, and other videos loaded on the page in a similar way, but from tsx components, don't have this problem, I apologize for the trouble and thank you for such a quick response.
